client/state/reader/follows/utils.js

```diff
@@ -5,6 +5,14 @@
 import { untrailingslashit } from 'lib/route';
 
 export function prepareComparableUrl( url ) {
-	const preparedUrl = url && untrailingslashit( url );
-	return preparedUrl && preparedUrl.replace( /^https?:\/\//, '' ).toLowerCase();
+	const unslashedUrl = url && untrailingslashit( url );
+	if ( ! unslashedUrl ) {
+		return null;
+	}
+	const preparedUrl = unslashedUrl.replace( /^https?:\/\//, '' ).toLowerCase();
+	if ( preparedUrl === 'www' || preparedUrl === 'www.' ) {
```

Comment:
maybe we should expand this to catch more invalid things? We could only let strings that contain `\w+\.\w+` (anything followed by a `.` followed by anything) through...

That would also catch things like `localhost` and weird singular names that only work on intranets

Reply:
Just remembered that we have `resemblesUrl` in `lib/url` for exactly this case. I'll pop that in.